packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -110,6 +111,7 @@ const initialState: SuiteState = {
         language: ensureLocale('en'),
         torOnionLinks: isWeb(),
         isCoinjoinExplanationHidden: false,
+        isCoinjoinCexWarningHidden: false,
```

Comment:
we should move it to the coinjoin reducer later (together with other coinjoin-related state), but let's not do it as part of this PR...